qrelu6 benchmarking

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #29427 Combining the ops in the qactivation benchmarks
* #29259 Adding longer tests for relu
* #29320 Adding short tests for qactivations
* **#29183 qrelu6 benchmarking**
* #29182 Creating a base benchmarking class for activations.

Differential Revision: [D18319499](https://our.internmc.facebook.com/intern/diff/D18319499)

Commit:
qrelu6 benchmarking

Commit:
Update on "qrelu6 benchmarking"


Differential Revision: [D18319499](https://our.internmc.facebook.com/intern/diff/D18319499)

Commit:
Update on "qrelu6 benchmarking"


Differential Revision: [D18319499](https://our.internmc.facebook.com/intern/diff/D18319499)

Commit:
Update on "qrelu6 benchmarking"


Differential Revision: [D18319499](https://our.internmc.facebook.com/intern/diff/D18319499)

Commit:
Update on "qrelu6 benchmarking"


Differential Revision: [D18319499](https://our.internmc.facebook.com/intern/diff/D18319499)

Commit:
Update on "qrelu6 benchmarking"


Differential Revision: [D18319499](https://our.internmc.facebook.com/intern/diff/D18319499)

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qactivation_test.py:
```diff
@@ -25,7 +25,7 @@
     permute_dims=(False, True),
     inplace=(False, True),
     dtype=(torch.quint8, torch.qint8, torch.qint32),
-    tags=('short',)
```
short is currently the default tag. If there is only one set of input shapes, you may want to mark it as 'short' to avoid passing tag_filter flag.

z-a-f replied:
I am adding long benchmarks in later PRs -- added short here in order not to forget :)

Commit:
Update on "qrelu6 benchmarking"


Differential Revision: [D18319499](https://our.internmc.facebook.com/intern/diff/D18319499)

Commit:
Update on "qrelu6 benchmarking"


Differential Revision: [D18319499](https://our.internmc.facebook.com/intern/diff/D18319499)

z-a-f commented:
This is squashed into #29182
